group/standard/variables.tf

```diff
@@ -41,6 +45,10 @@ variable "ebs_optimized" {
   default = false
 }
 
+variable "placement_tenancy" {
+  default = "default"
```

Comment:
Hmm, had a fun thought about limiting inputs. What if we converted some of our variables with a fixed set of inputs to maps that would do an identity map? It would probably push errors up from runtime to compile time.